### PR TITLE
Added Cols, Row and Utils Class

### PR DESCRIPTION
**Pull request by srujanponnur**

### CODE REVIEW

File: code/Cols.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        self.process_names(names)
+
+    def process_names(self, names):
+        for index, value in enumerate(names):
+            # column that starts with upper case alphabet is num else sym
+            col = Num(index, value) if value[0].isalpha() and value[0].isupper() else Sym(index, value)
+            Utils.push(self.all, col)
+            # print(self.all)
+            if not value.endswith(':'):  # columns ending with ':' are skipped
+                input_dict = self.x if any(value.endswith(i) for i in ['+', '!', '-']) else self.y
```
Comment (vishnuchalla) [inline code/Cols.py]: here string.find is different from string.endswith. Please replace it with something similar to find functionality. Lua tutorial here: https://www.tutorialspoint.com/string-find-function-in-lua

Comment (kanchan110295) [reply code/Cols.py]: Also, if the string finds +,!,- then it should be assigned to self.y and not self.x, https://github.com/timm/l5/blob/c0a035598d25234d5b8083fda94742d100fe7d3c/src/csv.lua#L144

Comment (kanchan110295) [reply code/Cols.py]: Add this line as well: https://github.com/timm/l5/blob/c0a035598d25234d5b8083fda94742d100fe7d3c/src/csv.lua#L145

File: code/Row.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Row:
```
Comment (vishnuchalla) [inline code/Row.py:1]: Remove this class. Lets add all these small utilities in a single Utils.py file.

Comment (srujanponnur) [reply code/Row.py:1]: Since its a separate class, let it be in a new file.

**Comment by vishnuchalla** [comment]
Merging it as there is only one pending discussion on string.find or string.endswith

**Pull request merged**